### DISCUSSION TITLE
Difference between hierarchy.current and hierarchy.selection?

### DISCUSSION THREAD

**Discussion by Marwa-Da** [molstar/molstar#915] [Q&A] [answered]
Hi all,
I'm interested in knowing the difference between plugin.managers.structure.hierarchy.current and plugin.managers.structure.hierarchy.selection.
I tried to dig deeper into the code related to them but I got lost after a while.

Thanks!

**Comment by dsehnal** [comment, marked as answer]
The selection comes into play when multiple structures are loaded at the same time and can be "edited" by this control:
![image](https://github.com/molstar/molstar/assets/21145776/d5a8e318-ed80-4550-8e9e-81d1f1c17b36)
